README.md

```diff
@@ -101,18 +101,10 @@ The example MGF file is available at [`sample_data/sample_preprocessed_spectra.m
 casanovo --mode=denovo --model='[PATH_TO]/pretrained_excl_mouse.ckpt' --peak_dir='sample_data' --config='path/to/config.yaml'
 ```
 Make sure you use the proper filepath to the `pretrained_excl_mouse.ckpt` file.
-    - Note: If you want to get the ouput CSV file in different location than the working directory, specify an alternative output location using the `--output` parameter.
+    - Note: If you want to get the output mzTab file in different location than the working directory, specify an alternative output location using the `--output` parameter.
 
 This job will take very little time to run (< 1 minute).
 
```

Comment:
**question (non-blocking):** Would it be good to provide a few example lines from the mzTab file?

Reply:
It will probably be useful to explain the format to some extent, but because there's a metadata section and then the PSM section, it's kind of verbose to include it in the README. I'd suggest to add a proper description when we set up a full documentation website on readthedocs. We already have #27 to remind us of it.